Controller/ContentManagement/JobController.php

```diff
@@ -130,15 +131,16 @@ public function startJobAction(Job $job, Request $request, JobService $jobServic
             return new JsonResponse('job already done');
         }
 
+        set_time_limit(0);
```

Comment:
Set time limit 0 Will not override php-fpm time limitations! And platform specific limitations imposed by the hosting environment.